src/color.rs

```diff
@@ -223,6 +223,20 @@ impl Color {
         a: NV_ONE,
     };
 
+    /// Creates a new color from 4 components.
+    ///
+    /// # Safety
+    ///
+    /// All values must be in 0..=1 range.
+    pub const unsafe fn from_unchecked_rgba(r: f32, g: f32, b: f32, a: f32) -> Self {
```

Comment:
Looking again, I'd prefer the name:
```suggestion
    pub const unsafe fn from_rgba_unchecked(r: f32, g: f32, b: f32, a: f32) -> Self {
```
The `_unchecked` as a suffix is much more idiomatic.
We can merge this once you respond to this comment.